Artifact/Views/ContentView.swift

```diff
@@ -1,11 +1,13 @@
 import SwiftUI
 
 struct ContentView: View {
+    @StateObject private var journeysViewModel = JourneysViewModel()
+    
     var body: some View {
         NavigationView {
```

Comment:
NavigationView is deprecated. You should consider using NavigationStack instead.